extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
Without fixedLayer, there shouldn't be a need to do anything with focus. Removing an element that has the focus from DOM, depending on the browser, should either keep the virtual cursor in the place where the removed DOM element was or move it to top of body as far as I know. (speculation though, worth testing).